CONTRIBUTING.md

```diff
@@ -0,0 +1,3 @@
+# Guide lines for contribuition in our gitbook!
+* [http://dev.docs.lightblue.io/development_process/README.html](http://dev.docs.lightblue.io/development_process/README.html)
+* Questions & suggestions,s end on nabble ->  http://forum.lightblue.io/ OR  http://dev.forum.lightblue.io/
```

Comment:
Fixed.

Thank you for both comments. 

The only reason that I put the http://forum.lightblue.io/ there is because the user that was about to create an issue about something related to usage could also see this page (just some more details for clarification: many projects in GIthub people interact exclusively by GIthub issues, even about usage or asking for a feature (that could even already have), so this group of people could assume that lightblue follow this pattern as well but when they see the new issue page they will have that notification box which would lead to the page rendered from the content of this file, so some user related links might be helpful as well (maybe even more helpful if we ask to search in the documentation and nabble and then create a thread on nabble) (I think the documentation also cover this point, but just wonder the best way to also inform the lightblue's users about those best practices )  )